WordPress/Classes/Utility/Media/MediaURLExporter.swift

```diff
@@ -77,10 +77,9 @@ class MediaURLExporter: MediaExporter {
     ///
     func exportURL(fileURL: URL, onCompletion: @escaping OnMediaExport, onError: @escaping OnExportError) -> Progress {
         // Verify the export is permissible
-        if let urlExportOptions = urlOptions,
-            !urlExportOptions.allowableFileExtensions.isEmpty,
-            let fileExtension = fileURL.typeIdentifierFileExtension {
-            if !urlExportOptions.allowableFileExtensions.contains(fileExtension) {
+        if let fileExtension = fileURL.typeIdentifierFileExtension {
+            // Check the default file types. We want to limit to supported types but mobile is not restricted to only allowed ones.
+            if !MediaImportService.defaultAllowableFileExtensions.contains(fileExtension) {
```

Comment:
Checked with @aforcier that Android does not appear to be using the `allowed_file_types` from the blog.

On iOS, we don't check the list for PHAssets but do for media which comes from a URL. This needs to change for Stories since we are uploading video which came from Photos (or WP Media) and then ended up re-exported with modifications.